services/device-registry-jdbc-quarkus/src/main/resources/application.properties

```diff
@@ -1,7 +1,13 @@
+quarkus.http.port=${vertx.health.port}
 # this needs to be configured at build time in order for native images to be able to log at TRACE level
 quarkus.log.min-level=TRACE
 quarkus.jackson.accept-case-insensitive-enums=true
 # fail deserialization of JSON objects sent by clients if they contain unexpected content
 quarkus.jackson.fail-on-unknown-properties=true
+quarkus.micrometer.binder-enabled-default= false
+quarkus.micrometer.binder.jvm=true
+quarkus.micrometer.binder.kafka.enabled=true
+quarkus.micrometer.binder.system=true
+quarkus.micrometer.binder.vertx.enabled=true
 quarkus.vertx.prefer-native-transport=true
 quarkus.vertx.resolver.cache-max-time-to-live: 0
```

Comment:
Off-topic, but the wrong ":" could be fixed here as well.

Reply:
nice catch :+1: